R/utils.R

```diff
@@ -86,3 +86,83 @@ check_date_time <- function(date_time, date_name = "start_date") {
   )
   as.character(parsed)
 }
+
+#' Get the credentials from environment variables, or set them manually
```

Comment:
Would make more succinct and refrain from using comma's in function titles:

```
Get login credentials
```

With the different credentials floating around (R Studio server, applications, DB) we should probably figure out how these can be called consistently.